sdk/keyvault/azure_security_keyvault_certificates/src/models.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+pub use crate::generated::models::*;
+use std::sync::LazyLock;
+
+/// Gets the default self-signed [`CertificatePolicy`].
+pub static DEFAULT_POLICY: LazyLock<CertificatePolicy> = LazyLock::new(|| CertificatePolicy {
```

Comment:
@RickWinter @chlowell this change is a bit orthogonal, but:

1. All other languages but Go have something similar. Even the `az` CLI has this.
2. It greatly simplifies examples.

Ya'll okay with it?

Reply:
Nit: We could have this code in each example or an example_helper class.

I'm thinking about location b/c I wonder how much this type gets used by real world scenarios

Reply:
I suppose `DEFAULT_CERTIFICATE_POLICY` would be better, sure. Other languages just often use some variation of "default" e.g., in .NET it's `CertificatePolicy.Default` (static read-only property), but something like `CertificatePolicy::default()` for Rust has a different meaning i.e., the `Default::default()` implementation.

As for making this only available to doc tests,

1. There is a known issue with including things only for `cfg(any(test, doctest))` but,
2. For using in copy-and-paste / "getting started" examples, it'd be a disservice to customers to use something that doesn't actually exist when they copy the code.

Reply:
Removed the public API and inlined the definition in samples per internal discussions.